app/web/src/newhotness/Explore.vue

```diff
@@ -802,49 +795,117 @@ const sortedAndGroupedComponents = computed(() => {
 // THE SEARCH BAR AND FILTERING
 const searchString = ref<string>("");
 const debouncedSearchString = ref<string>("");
-const computedSearchString = computed(() => debouncedSearchString.value);
 
 // send this down to any components that might use it
-provide("SEARCH", computedSearchString);
-
-watch(
-  () => [debouncedSearchString.value, componentList.value],
-  () => {
-    if (!debouncedSearchString.value) {
-      filteredComponents.splice(0, Infinity, ...componentList.value);
-      return;
+provide("SEARCH", debouncedSearchString);
+
+/**
+ * Search string, split into terms.
+ *
+ * MyComponent vpcId:vpc-123 AWS::EC2::Subnet parses to:
+ *
+ *     {
+ *       attrSearchTerms: [ { key: "vpcId", value: "vpc-123" } ],
+ *       knownSearchTerms: [ { key: "schema", value: "AWS::EC2::Subnet" } ],
+ *       fuzzyTerms: [ "MyComponent", "AWS::EC2::Subnet" ]
+ *     }
+ */
+const debouncedSearchTerms = computed(() => {
+  const attrSearchTerms = new Array<{ key: string; value: string }>();
+  const knownSearchTerms = new Array<{ key: "schema"; value: string }>();
```

Comment:
i feel like we'll have a better chance of making the fuzzy search stricter/less pervasive than special casing a list of all potential exact matches but 🤷

Reply:
Discussed in zoom to make sure we had the right scope--we added category in to handle this.